core/src/modules/statusReporters.ts

```diff
@@ -601,7 +601,8 @@ export namespace FinalSummaryReporters {
       });
       const allRecords = await GrouparooRecord.count();
 
-      const recordData = {
+      const recordData: RecordData = {
+        name: null,
```

Comment:
Unrelated, but noticed that having this `undefined` made the string "undefined" show up in the final status reporter as the name for the section. This was previously explicitly set to null, but removed (likely unintentionally) as part of #2747

Reply:
Whoops!